lib/common/alarm/AlarmFactory.ts

```diff
@@ -549,32 +568,56 @@ export class AlarmFactory {
       );
     }
 
+    // apply metric math for minimum metric samples
+
+    if (props.minSampleCountToEvaluateDatapoint) {
+      if (adjustedMetric instanceof MathExpression) {
+        throw new Error(
+          "minSampleCountToEvaluateDatapoint is not supported for MathExpressions"
```

Comment:
I would add a sentence that if you already use metric math you can extend your expression to do the sample count evaluation.

Reply:
good idea, done